lib/src/common/model/search.dart

```diff
@@ -7,7 +7,7 @@ class SearchResults<T> {
   @JsonKey(name: "incomplete_results")
   bool incompleteResults;
 
-  List<T> items;
+  List<T> items = <T>[];
 
   static SearchResults fromJSON(
```

Comment:
Want to keep converting these to `fromJson` and using json_serializable to auto-generate the serialization / deserialization code?